reactive/kubernetes_worker.py

```diff
@@ -965,25 +965,19 @@ def render_and_launch_ingress():
         context['default_ssl_certificate_option'] = default_certificate_option
     context['ingress_image'] = config.get('nginx-image')
     if context['ingress_image'] == "" or context['ingress_image'] == "auto":
-        if registry_location:
-            nginx_registry = registry_location
+        if context['arch'] == 'ppc64el':
+            # multi-arch image doesn't include ppc64le, have to use an older version
+            context['ingress_uid'] = '33'
+            context['ingress_image'] = '/'.join([
+                registry_location or 'quay.io',
+                'kubernetes-ingress-controller/nginx-ingress-controller-ppc64le:0.20.0',
+            ])
         else:
-            nginx_registry = 'quay.io'
-        images = {
-            'amd64': 'kubernetes-ingress-controller/nginx-ingress-controller-amd64:0.31.1',  # noqa
-            'arm64': 'kubernetes-ingress-controller/nginx-ingress-controller-arm64:0.31.1',  # noqa
-            's390x': 'kubernetes-ingress-controller/nginx-ingress-controller-s390x:0.20.0',  # noqa
-            'ppc64el': 'kubernetes-ingress-controller/nginx-ingress-controller-ppc64le:0.20.0',  # noqa
-        }
-        # NB: ingress >= 0.27 switched to alpine, where www-data uid is now 101
-        # https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.27.0
-        if context['arch'] == 'amd64' or context['arch'] == 'arm64':
             context['ingress_uid'] = '101'
-        else:
-            context['ingress_uid'] = '33'
-        context['ingress_image'] = '{}/{}'.format(nginx_registry,
-                                                  images.get(context['arch'],
-                                                             images['amd64']))
+            context['ingress_image'] = '/'.join([
+                registry_location or 'us.gcr.io',
+                'k8s-artifacts-prod/ingress-nginx/controller:v0.34.1',
```

Comment:
I wonder if this is going to need to be `us.gcr.io/k8s-artifacts-prod` as the registry (if no `registry_location`), and if so, the image will be `ingress-nginx/controller:v0.34.1`.  Let's see what comes out of the cdk-addons build job to know how the image looks in rocks.

Reply:
The path, after the fix in charmed-kubernetes/cdk-addons#192 lands, will be `"image: {{ registry|default('k8s.gcr.io') }}/k8s-artifacts-prod/ingress-nginx/controller:v0.34.1"` so this PR should be fine.